spec/translator_spec.rb

```diff
@@ -15,12 +16,35 @@
 
   describe "options" do
     describe "obfuscate: [fields]" do
-      it 'scrambles an specific field' do
-        translator = Polo::Translator.new([finder], Polo::Configuration.new(obfuscate: [:email]))
-        netto = translator.instances.first
+      let(:translator) { Polo::Translator.new([finder], Polo::Configuration.new(obfuscate: obfuscated_fields)) }
+      let(:netto) { netto = translator.instances.first }
 
-        expect(netto.email).to_not be_nil
-        expect(netto.email).to_not eq('nettofarah@gmail.com')
+      context "with obfuscated field with no specified strategy" do
+        let(:obfuscated_fields) {{email: nil }}
+        it "shuffles characters in field" do
+          expect(netto.email).to_not be_nil
+          expect(netto.email.length).to eq email.length
+          expect(sorted_characters(netto.email)).to eq sorted_characters(email)
+        end
+      end
+
+      context "with obfuscated field with integer strategy" do
+        let(:obfuscated_fields) {{email: lambda { |_| 42 } } }
```

Comment:
Maybe we make this a little more clear here.
I get what's going one, but maybe "integer strategy" is too generic.